x/chainlink/client/cli/txFeed.go

```diff
@@ -6,27 +6,32 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/spf13/cobra"
+	"strings"
 )
 
-func CmdSubmitFeed() *cobra.Command {
+func CmdSubmitFeedData() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "submit-feedData [feedId] [feedData]",
+		Use:   "submit-feedData [feedId] [feedData] [signatures]",
```

Comment:
need long description too. can just add TODO comment for now

Reply:
yeah good idea